loom/engine/bindings/loom/lmApplicationTasks.cpp

```diff
@@ -36,11 +36,26 @@ extern "C"
 {
 
 atomic_int_t gLoomTicking = 1;
+atomic_int_t gLoomPaused = 0;
 
 void loom_tick()
 {
-    if (atomic_load32(&gLoomTicking) < 1) return;
-    
+    if (atomic_load32(&gLoomTicking) < 1)
+    {
+
+        // Signal that the app has really stopped execution
+        lmLogWarn(gTickLogGroup, "Paused: %d", atomic_load32(&gLoomPaused));
```

Comment:
Won't this print every tick while the app is paused? Assuming it gets scheduled time.